CHANGES.rst

```diff
@@ -18,6 +18,8 @@ New Features
 
 - ``astropy.io.fits``
 
+  - Checking available disk space before writing out file. [#5500, #4065]
```

Comment:
Oh noes! I thought you fixed this but maybe it was lost in rebase. Can you please submit a small PR to fix this change log? If not, maybe someone with write priviledge to master can just change it there?